src/index.html

```diff
@@ -1,14 +1,41 @@
+<!-- #region head-->
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Stop watch</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Stop watch</h1>
+    <div class="watch">
+      <div class="watch__wrapper watch__wrapper--marks">
+        <div class="watch__marks watch__marks--1"></div>
+        <div class="watch__marks watch__marks--2"></div>
```

Comment:
![image](https://github.com/mate-academy/layout_stop-watch/assets/96409563/c088355b-e2a3-414c-b20c-d93ef051d4de)

Reply:
It's just like a background, I couldn't come up with better.